TanukiKitTests/ProjectTests.swift

```diff
@@ -0,0 +1,112 @@
+import XCTest
+import TanukiKit
+
+class ProjectTests: XCTestCase {
+
+    // MARK: Actual Request tests
+
+    // TODO: Change to projects.
+    func testGetProjectsCOM() {
+        let session = TanukiKitURLTestSession(expectedURL: "https://gitlab.com/api/v3/projects?access_token=12345&archived=&order_by=created_at&page=1&per_page=20&search=&simple=&sort=desc&visibility=", expectedHTTPMethod: "GET", jsonFile: "Repositories", statusCode: 200)
+        let config = TokenConfiguration("12345")
+        _ = TanukiKit(config).repositories(session) { response in
+            switch response {
+            case .success(let repos):
+                XCTAssertEqual(repos[0].name, "Diaspora Project Site")
+            case .failure(let error):
+                XCTAssert(false, "❌ Should not retrieve an error –> (\(error))")
+            }
+        }
+        XCTAssertTrue(session.wasCalled)
+    }
+
+    // TODO: Change to projects.
+    func testGetProjectsEECE() {
```

Comment:
This method is calling `.repositories(session)` so it would make sense to name this method `func testGetRepositoriesEECE()`